public/shared/messages.js

```diff
@@ -31,10 +31,30 @@ const ERROR_DELETING_DATASET_MESSAGE =
 const SUCCESS_DELETING_DATASET_MESSAGE = 'The dataset was deleted successfully';
 const SUCCESS_DELETING_RESULT_MESSAGE =
   'The result dataset was deleted successfully';
-  const SUCCESS_DELETING_ALGORITHM_MESSAGE =
+const SUCCESS_DELETING_ALGORITHM_MESSAGE =
   'The algorithm was deleted successfully';
 const ERROR_DELETING_RESULT_MESSAGE =
   'There was an error deleting the result dataset';
+const ERROR_PYTHON_NOT_INSTALLED_MESSAGE =
+  'Python is not installed on your system';
+const buildPythonWrongVersionMessage = (version) =>
+  `Your system's python version (${version}) is not supported`;
+const ERROR_CHECKING_PYTHON_INSTALLATION_MESSAGE =
+  'There was an error checking the installation of python';
```

Comment:
Maybe: _There was an error verifying the installation of Python on your system. Try restarting the application. If the error persists, try reinstalling Python._